### ISSUE TITLE
How can I do a ModelConverter which involves generics? 

### ISSUE THREAD

**Issue by tzimisce012** [swagger-api/swagger-core#4183]
So I just want to transform spring `Page<T>` so I just display some fields with different names. I checked [this](https://github.com/springdoc/springdoc-openapi/blob/master/springdoc-openapi-common/src/main/java/org/springdoc/core/converters/PageableOpenAPIConverter.java) converter and I tried to do the same. The problem comes when I have to reach they binding from my source class and try to bind that to my custom class. Eventually I have a class like this:

```java
public class CustomPage<T> {
   private Integer total;
   private Integer count;
   private List<T> list;
}
```
but I am not able to "assign" the bindingtype from spring `Page` to my `CustomPage`

My problems is so far the same than this one: https://github.com/springdoc/springdoc-openapi/issues/981

**Comment by amateos-stratio** [comment]
Yes, I have the same problem and it would be interesting to know how to handle this problem, pls

**Comment by pdavideb** [comment]
I am also interested in this, please

**Comment by MichakrawSB** [comment]
Closing this issue as part of our https://github.com/swagger-api/swagger-core/issues/5074.
This issue is outdated and/or inactive for a long time, and we’re not able to revisit it meaningfully.

If this problem is still relevant, please open a new issue and reference this one for context.
Thank you for your understanding!
